package.json

```diff
@@ -38,6 +38,9 @@
     "proxyquire": "^1.7.4",
     "shelljs": "^0.6.0",
     "sinon": "^1.17.3",
-    "sinon-chai": "^2.8.0"
+    "sinon-chai": "^2.8.0",
+    "yeoman-assert": "^2.2.1",
```

Comment:
I think these are not necessary.  I'm able to do a fresh checkout and get everything running without these dependencies.

```
git clone git@github.com:Shopify/javascript.git
cd javascript
script/setup
npm test
cd packages/shopify-codemod
npm test
```